src/app/routes/onDemandAudio/tempData/podcastExternalLinks/persian.js

```diff
@@ -50,6 +50,12 @@ const externalLinks = {
           'https://podcasts.apple.com/gb/podcast/%D8%AF%D8%A7%D8%B3%D8%AA%D8%A7%D9%86-%D8%A7%D9%86%D9%82%D9%84%D8%A7%D8%A8-%D8%A8%D9%87-%D8%B1%D9%88%D8%A7%DB%8C%D8%AA-%D8%A8%DB%8C-%D8%A8%DB%8C-%D8%B3%DB%8C/id1451707438',
         linkType: 'apple',
       },
+      {
+        linkText: 'Castbox',
+        linkUrl:
+          'https://castbox.fm/channel/%D9%85%D8%B3%D8%AA%D9%86%D8%AF-id2007735?country=gb',
```

Comment:
Ooh this is interesting! I clicked on the link, and I can see that it is asking to claim ownership - is this something we should consider doing? 

![image](https://github.com/user-attachments/assets/f36a0a2f-a86c-4eae-a61f-502d7e6a4525)